### PR TITLE
S1t front end

### PR DESCRIPTION
**Pull request by MattW-R**

### CODE REVIEW

File: client/src/Component/AllCarParks/AllCarParks.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    display: flex;
+    justify-content: space-between;
+}
+
+.all-car-parks-header h1 {
+    height: 36px;
+    width: 150px;
+    color: #6E6E6E;
+    font-family: Prompt, sans-serif;
+    font-size: 24px;
```
Comment (emmatrain) [inline client/src/Component/AllCarParks/AllCarParks.css]: do we need to use REM for responsiveness?

File: client/src/Component/AllCarParks/AllCarParks.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            .then(res => res.json())
+            .then(data => {
+                if (data.success) {
+                    setCarParks(displayCarParks(data.data))
+                } else {
+                    setCarParks(<NoCarParks />)
+                }
+            })
+            .catch(() => setCarParks(<NoCarParks />))
+        // eslint-disable-next-line
```
Comment (emmatrain) [inline client/src/Component/AllCarParks/AllCarParks.js]: does this need deleting out or is it for the future in some capacity?

File: client/src/Component/CarPark/CarPark.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+.card h3, .card p {
+    letter-spacing: 0;
+    color: #6E6E6E;
+    font-family: Prompt, sans-serif;
+    margin-left: 16px;
+}
+
+.card h3 {
+    height: 24px;
+    font-size: 16px;
```
Comment (emmatrain) [inline client/src/Component/CarPark/CarPark.css]: rem?

File: client/src/Component/NoCarParks/NoCarParks.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.no-car-parks {
+    letter-spacing: 0;
+    color: #6E6E6E;
+    font-family: Prompt, sans-serif;
+    margin-left: 16px;
+    height: 21px;
+    font-size: 14px;
```
Comment (emmatrain) [inline client/src/Component/NoCarParks/NoCarParks.css]: rem if that's what we're doing?

File: client/src/Component/AllCarParks/AllCarParks.js (hunk 2 above)
Comment (MattW-R) [reply client/src/Component/AllCarParks/AllCarParks.js]: This is to shut react up ^.^

**Pull request merged**